### DISCUSSION TITLE
Is there any API to transfer items between field-array?

### DISCUSSION THREAD

**Discussion by B-Esmaili** [react-hook-form/react-hook-form#7380] [General] [answered]
I want to implement a drag and drop between two list of items constructed using `useFieldArray`. I couldn't find any API like 'move' which I can use to transfer items between two list. Is there any API to do this?

**Comment by bluebill1049** [comment, marked as answer]
we don't have such API to correct swap or move field array(s) with different field array name.

**Reply by B-Esmaili** [reply]
Ok. Thanks for your answer.

**Comment by arimendelow** [comment]
@bluebill1049 do you have any plans to create such an API?
